geonode/geoserver/helpers.py

```diff
@@ -638,13 +640,29 @@ def set_attributes(layer, overwrite=False):
             attribute_map = []
 
     elif layer.storeType in ["dataStore", "remoteStore", "wmsStore"]:
-        dft_url = re.sub("\/wms\/?$",
+        dft_url = ''
+        if 'geonode.geoserver' in settings.INSTALLED_APPS:
```

Comment:
This worries me - we should chat to @jjohnson about how we can get geoserver stuff better separated out of the genome app otherwise we are going to end up with the code littered with if branches depending on whether we have geoserver backend or not. I think we should assume that more backends are coming - e.g. a mapserver one and we should pave the way for a clean backend separation.